src/test/suite/fixtures/recorded/languages/csharp/changeCallee.yml

```diff
@@ -0,0 +1,23 @@
+languageId: csharp
```

Comment:
@AndreasArvidsson I know you originally were pushing to include `new`, but I believe yesterday we had a use case where you wanted to just target `TestMotorcycle` here, without the `new`.  WDYT?

Reply:
For type I definitely think new should be excluded for callee one could make an argument for both. If you prefer without new I'm happy with that.

Reply:
Would personally prefer it without the new but I could also see the argument either way.

Reply:
Ok let's drop the `new` 👍